lab_3.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+import sys
+import boto3
+import datetime
+
+s3 = boto3.client('s3')
+
+def main(argv):
+    cmd = argv
+    log_file_name = datetime.datetime.now().strftime("%m_%d_%Y") + "_logfile"
+    kickoff_subprocess(cmd, log_file_name)
+    upload_output_to_S3(log_file_name)
+
+def kickoff_subprocess(cmd, log_file_name):
+    process = subprocess.call(cmd, shell=True)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

subprocess call with shell=True identified, security issue. https://bandit.readthedocs.io/en/latest/plugins/b602_subprocess_popen_with_shell_equals_true.html